scanpipe/templates/scanpipe/project_detail.html

```diff
@@ -244,6 +271,10 @@ <h3 class="title is-4 has-text-centered mb-3">
     // Packages
     makeChart("package_types", "#package_type_chart", "Package\nType");
     makeChart("package_licenses", "#package_license_chart", "Package\nLicense\nExpression");
+    // Dependencies
+    makeChart("dependency_is_runtime", "#dependency_is_runtime_chart", "Dependency\nis_runtime");
```

Comment:
What about "Runtime Dependencies", "Optional Dependencies", and "Resolved Dependencies" as labels?

Reply:
I've updated the dependency donut chart labels